### ISSUE TITLE
Add language models

### ISSUE THREAD

**Issue by darsnack** [FluxML/ML-Coordination-Tracker#3]
Add standard language models.

**Comment by SomTambe** [comment]
@darsnack I would like to help with adding functionality related to data loading and pre-processing. I have worked on pytorch only yet, and I had faced some problems while loading 3d-vision datasets. To resolve my issues, I had to modify some of the source code of some dataset classes, and due to that I had a peek at the inner functionality of pytorch. I have used Julia since 7 months now, I believe I could help with this task.
Kindly update me with what you plan to do, I would be more than happy to add functionality 😀  .
